bundles/org.openhab.binding.gridbox/src/main/resources/OH-INF/i18n/gridbox.properties

```diff
@@ -0,0 +1,33 @@
+# add-on
+
+addon.gridbox.name = GridBox Binding
+addon.gridbox.description = Get Information from the Viessmann GridBox
+
+# thing types
+
+thing-type.gridbox.gridbox.label = GridBox
+thing-type.gridbox.gridbox.description = GridBox Thing
+
+# thing types config
+
+thing-type.config.gridbox.gridbox.email.label = E-Mail
+thing-type.config.gridbox.gridbox.email.description = E-Mail address used to log in to the GridBox API (https://mygridbox.viessmann.com/)
+thing-type.config.gridbox.gridbox.password.label = Password
+thing-type.config.gridbox.gridbox.password.description = Password to access the GridBox API
+thing-type.config.gridbox.gridbox.refreshInterval.label = Refresh Interval
+thing-type.config.gridbox.gridbox.refreshInterval.description = Interval the device is polled in sec.
+
+# channel types
+
+channel-type.gridbox.rate.label = Rate
+channel-type.gridbox.stateOfCharge.label = State of Charge
```

Comment:
This indicates the i18n still needs to be run after all the channel id renaming.

Reply:
I did not rename the channel-type id yet...